pkg/agent/datapath/multiBridgeDatapath.go

```diff
@@ -849,11 +849,6 @@ func (datapathManager *DpManager) AddEveroutePolicyRule(rule *EveroutePolicyRule
 			return nil
 		}
 		log.Infof("Rule already exists. update old rule: {%+v} to new rule: {%+v} ", ruleEntry.EveroutePolicyRule, rule)
-
-		// clear CT flow while updating from "allow" to "deny"
-		if ruleEntry.EveroutePolicyRule.Action == EveroutePolicyAllow && rule.Action == EveroutePolicyDeny {
```

Comment:
Is there a possibility the rule has no changed, and don't need clean the ct flow?

Reply:
Same rule will return in L846